css/index.css

```diff
@@ -112,8 +112,7 @@ footer {
 /*Patriot web title*/
 .greetingHeader {
     font-size: 75px;
-    /*font-family: URW Chancery L, cursive;*/
-    font-family: "Brush Script MT Bold", cursive;
+    font-family: "Brush Script MT Bold";
```

Comment:
there needs to be a generic default for the font-family; also, the sub-header and the picture labels are still in the cursive script